nix/gce-images.nix

```diff
@@ -0,0 +1,7 @@
+let self = {
```

Comment:
I think we can have this file in nixpkgs somhow like `ec2-amis.nix` and export an attrs:
```
{ imageFamily = "nixos-20-03";
  project = "nixos-org";
}
```
So that updates of images are automatic and without changes to nixops-gce.